### PR TITLE
Recorder UI Added

### PR DESCRIPTION
**Pull request by arjxn-py**
### UI of Recorder Added using Tailwind CSS.
image

### CODE REVIEW

File: curio/app/src/components/App.js [hunk 1]
```diff
@@ -1,22 +1,17 @@
...
-// import { Recorder } from 'react-voice-recorder';
+// import { Recorder , myRecorder} from 'react-voice-recorder';
+import {button} from './recorder'
 // import ReactPlayer from 'react-player/youtube'
 import './App.css'
 
 function App () {
   return(
-    <h1>Hello World</h1>
+    // <h1>Hello World</h1>
```
Comment (Saransh-cpp) [inline curio/app/src/components/App.js]: Commenting out the not required code is always a bad idea.
```suggestion
```

File: curio/app/src/components/App.js [hunk 2]
```diff
@@ -1,22 +1,17 @@
 import React from 'react';
-// import { Recorder } from 'react-voice-recorder';
+// import { Recorder , myRecorder} from 'react-voice-recorder';
+import {button} from './recorder'
 // import ReactPlayer from 'react-player/youtube'
```
Comment (Saransh-cpp) [inline curio/app/src/components/App.js:4]: I am guessing you require these commented out imports in the future? If not, then they can be removed!

File: curio/app/src/components/recorder.js [hunk 3]
```diff
@@ -1,11 +1,34 @@
-
-import {Recorder} from 'react-voice-recorder'
+// import {Recorder} from 'react-voice-recorder'
```
Comment (Saransh-cpp) [inline curio/app/src/components/recorder.js:1]: Ditto!

File: curio/app/src/components/recorder.js [hunk 4]
```diff
@@ -1,11 +1,34 @@
...
+      </form>
+    </>
     );
   }
+
+
   
-  export default myRecorder;
\ No newline at end of file
+  // export default myRecorder;
```
Comment (Saransh-cpp) [inline curio/app/src/components/recorder.js:34]: Ditto for the comment!

Also, it is a good practice to leave an empty line at the end of every file.

image

This shows that your file does not have an empty line at the end.

**Review by Saransh-cpp — APPROVED** [review]
Nice work, @arjxn-py! Some very minor comments below. Note, these comments can be ignored in a personal project, but these comments are important for user-facing projects.

File: curio/app/src/components/App.js (hunk 2 above)
Comment (arjxn-py) [reply curio/app/src/components/App.js:4]: Yes, there are high chances of usage of these imports in the future. But yes, if they would be left unused they would be removed.

File: curio/app/src/components/recorder.js (hunk 3 above)
Comment (arjxn-py) [reply curio/app/src/components/recorder.js:1]: Making sure that these points are resolved in the next commit.

File: curio/app/src/components/recorder.js (hunk 4 above)
Comment (arjxn-py) [reply curio/app/src/components/recorder.js:34]: Thank You So Much @Saransh-cpp for your effective suggestions, will definitely take care of all the things I learnt in the future.

**Review by arjxn-py — COMMENTED** [review]
Have Made Some Suggested Changes.

**Pull request merged**